packages/logger/lib/logger.js

```diff
@@ -1,6 +1,6 @@
 const pino = require('pino').default;
 const serializeError = require('@dotcom-reliability-kit/serialize-error');
-const { default: structuredClone } = require('@ungap/structured-clone');
+const { default: clone } = require('@ungap/structured-clone');
```

Comment:
**note:** I renamed this to avoid confusion. In Node.js 18+ `structuredClone` is a global which doesn't exactly match the method signature of ungap's structured-clone module. I can imagine this confusing us a lot later.